Assets/Scripts/UI/CameraIntMsg.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class CameraIntMsg : MonoBehaviour {
+
+    public GameObject Camera;
+    public GameObject ArrowsSide;
+    public GameObject ArrowsUp;
+    public Controller2 isghostMode;
+    private bool onlyOnce;
+
+    // Use this for initialization
+    void Start()
+    {
+        onlyOnce = false;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.C))
```

Comment:
Let's use the input defined in the input manager https://docs.unity3d.com/Manual/class-InputManager.html instead of hardcoded keys. This way we can add controller support if need be.